FireRender.Maya.Src/InstancerMASH.cpp

```diff
@@ -73,7 +92,9 @@ void InstancerMASH::OnPlugDirty(MObject& node, MPlug& plug)
 	{
 		for (auto o : m_instancedObjects)
 		{
-			o.second->setVisibility(false);
+			auto it = o.second.begin();
+			for (; it != o.second.end(); ++it)
```

Comment:
why iterator declaration is outside loop?

Reply:
Has this been answered?

Reply:
Oh. Updated.
No reason to declare iterator outside the loop.
Changed so it would be the same as everywhere (iterator declared inside for statement)